engines/pytorch/pytorch-engine/src/test/java/ai/djl/pytorch/jni/IValueUtilsTest.java

```diff
@@ -42,4 +42,42 @@ public void getInputsTestTupleSyntax() {
             Arrays.stream(iValues).forEach(IValue::close);
         }
     }
+
+    @Test
+    public void testMapOfTensor() {
+        try (NDManager manager = NDManager.newBaseManager()) {
+            NDArray array1 = manager.zeros(new Shape(1));
+            array1.setName("input1.key1");
```

Comment:
minor: add some comments to explain why use `input1` as prefix? Like group all keys with same prefix into one Map?